padle/padle.py

```diff
@@ -0,0 +1,801 @@
+            return True
+        if not await self.config.user(ctx.author).start():
+            await ctx.send(f"You have not started the game of PADle yet, try `{prefix}padle start`!")
+            return True
+        if await self.config.user(ctx.author).done():
+            await ctx.send("You have already played today's PADle!")
+            return True
+        return False
+
+    @padle.command()
+    async def guess(self, ctx, *, guess):
+        """Guess a card for the daily PADle"""
+        if await self.ensure_active_game(ctx):
+            return
+        dbcog = await self.get_dbcog()
+
+        guess_monster = await dbcog.find_monster(guess, ctx.author.id)
+        if guess_monster is None:
+            close_menu = ClosableEmbedMenu.menu()
+            props = PADleMonsterConfirmationViewProps("Monster not found, please try again.")
+            query_settings = await QuerySettings.extract_raw(ctx.author, self.bot, guess)
+            state = ClosableEmbedViewState(ctx.author.id, ClosableEmbedMenu.MENU_TYPE, guess,
+                                           query_settings, PADleMonsterConfirmationView.VIEW_TYPE, props)
+            await close_menu.create(ctx, state)
+            return
+
+        m_embed = EmbedView(
+            EmbedMain(
+                title=MonsterHeader.menu_title(guess_monster).to_markdown(),
+                description="Did you mean this monster?"),
+            embed_thumbnail=EmbedThumbnail(
+                MonsterImage.icon(guess_monster.monster_id))).to_embed()
+        did_you_mean_msg = await ctx.send(embed=m_embed)
+        confirmation = await get_user_reaction(ctx, did_you_mean_msg, YES_EMOJI, NO_EMOJI, timeout=20)
+        if confirmation is None:
+            return await send_cancellation_message(ctx, "Confirmation timeout.")
+        if confirmation == NO_EMOJI:
+            return await send_cancellation_message(ctx, "Please guess again.")
+        if not await self.config.user(ctx.author).start():
+            return await send_cancellation_message(ctx,
+                                                   "You have not started a game yet! The PADle may have just expired.")
+        if await self.config.user(ctx.author).done():
+            return await send_cancellation_message(ctx, "You have already finished today's PADle!")
+        async with self.config.user(ctx.author).todays_guesses() as todays_guesses:
+            todays_guesses.append(guess_monster.monster_id)
+        monster = dbcog.get_monster(int(await self.config.padle_today()))
+        cur_day = await self.config.num_days()
+        todays_guesses = await self.config.user(ctx.author).todays_guesses()
+        async with self.config.user(ctx.author).all_guesses() as all_guesses:
+            all_guesses[str(cur_day)] = todays_guesses
+
+        channel = self.bot.get_channel(await self.config.user(ctx.author).channel_id())
+        if channel is not None:
+            try:
```

Comment:
instead of the
```py
try:
    # some code
except SpecificException:
    pass
```

you can use

```py
with suppress(SpecificException):
    # some code
````

`suppress` needs to be imported from `contextlib` if you wanna do this